api/CSSKeyframeRule.json

```diff
@@ -4,12 +4,26 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSKeyframeRule",
         "support": {
-          "chrome": {
-            "version_added": true
-          },
-          "chrome_android": {
-            "version_added": "45"
-          },
+          "chrome": [
+            {
+              "version_added": "31"
```

Comment:
Confirmed by looking up the date of https://chromium.googlesource.com/chromium/src/+/5a98a4be279589ff8b93bf32a34d38f380de4946 in https://www.chromium.org/developers/calendar.